UI/MeterBrowseWnd.cpp

```diff
@@ -327,8 +327,12 @@ void MeterBrowseWnd::UpdateEffectLabelsAndValues(GG::Y& top) {
     if (!maybe_info_vec)
         return;
 
+    // helpers for combining building effects into a single line
+    std::vector<std::string> combinedNames;
+    float combinedChange = 0.0;
```

Comment:
prefer `combined_names` and `combined_change` format

put `0.0f` for float constants